docs/installation-vagrant.rst

```diff
@@ -0,0 +1,119 @@
+Kitsune via Vagrant
+================
+
+Core developers run Kitsune in a `Vagrant`_-managed virtual machine so we can run
+the entire Kitsune stuck.
+If you're on Mac OS X or Linux and looking for a quick way to get started, you
```

Comment:
This only talks about Mac OS X and Linux, but then has Windows instructions below. It isn't consistent.